bokehjs/src/coffee/models/plots/gmap_plot_canvas.coffee

```diff
@@ -4,44 +4,45 @@ import {PlotCanvas, PlotCanvasView} from "./plot_canvas"
 import * as p from "../../core/properties"
 import {defer} from "../../core/util/callback"
 
+load_google_api = (callback, api_key) ->
+  if not window.google?.maps?
+    window._bokeh_gmap_callback = () -> callback()
```

Comment:
I can restore the more complicated "list of callbacks" if you think it's a priority.